src/components/GlobalPreferences/Notifications/ManageNotifications.js

```diff
@@ -0,0 +1,195 @@
+            token={token}
+            isFetchingSubscriptions={isFetching}
+            subscriptions={subscriptions}
+          />
+        }
+        secondary={
+          <React.Fragment>
+            <Box heading="Signed In With Email">
+              {email}
+              <Button
+                css={`
+                  margin-top: ${2 * GU}px;
+                `}
+                wide
+                onClick={onLogout}
+              >
+                Sign Out
+              </Button>
+            </Box>
+            <DeleteAccount
+              onApiError={setApiError}
+              token={token}
+              onLogout={onLogout}
+            />
+          </React.Fragment>
+        }
+      />
+      {(apiError || subscriptions.length > 0) && (
+        <SubscriptionsTable
+          apps={apps}
+          apiError={apiError}
+          onApiError={setApiError}
+          authToken={token}
+          subscriptions={subscriptions}
+          fetchSubscriptions={fetchSubscriptions}
+          isFetchingSubscriptions={isFetching}
+        />
+      )}
+    </React.Fragment>
+  )
+}
+
+ManageNotifications.propTypes = {
+  apps: PropTypes.arrayOf(AppType).isRequired,
+  dao: PropTypes.string,
+  email: PropTypes.string,
+  onLogout: PropTypes.func,
+  onServiceUnavailable: PropTypes.func,
+  token: PropTypes.string,
+}
+
+function DeleteAccount({ token, onLogout, onApiError }) {
+  const [isFetching, setIsFetching] = useState(false)
+  const [isAccountDeleted, setIsAccountDeleted] = useState(false)
+  const theme = useTheme()
+
+  const handleDeleteAccount = useCallback(async () => {
+    try {
+      setIsFetching(true)
+      await deleteAccount(token)
+      localStorage.removeItem(NOTIFICATION_SERVICE_TOKEN_KEY)
+      localStorage.removeItem(NOTIFICATION_SERVICE_EMAIL_KEY)
+      setIsAccountDeleted(true)
+      onLogout()
+    } catch (e) {
+      onApiError(e)
+    }
+    setIsFetching(false)
+  }, [token, onLogout, onApiError])
+
+  const [isModalOpen, setIsModalOpen] = useState(false)
+
+  const onClick = useCallback(() => {
+    setIsModalOpen(true)
+  }, [setIsModalOpen])
+
+  const onCloseModal = useCallback(() => {
+    setIsModalOpen(false)
+  }, [setIsModalOpen])
+
+  const onModalConfirm = useCallback(() => {
+    setIsModalOpen(false)
+    handleDeleteAccount()
+  }, [handleDeleteAccount, setIsModalOpen])
+
+  return (
+    <React.Fragment>
+      <DeleteAccountConfirmationModal
+        visible={isModalOpen}
+        onConfirm={onModalConfirm}
+        onClose={onCloseModal}
+      />
+      <Box heading="Email Notification Data">
+        <Button onClick={onClick}>
+          {isFetching ? (
+            <LoadingRing />
+          ) : isAccountDeleted ? (
+            <IconCheck />
+          ) : (
+            <IconTrash
+              css={`
+                color: ${theme.negative};
+                margin-right: ${GU}px;
+              `}
+            />
+          )}
+          Delete your email
```

Comment:
![image](https://user-images.githubusercontent.com/3072458/64102269-e34c4080-cd6f-11e9-98cc-f43b2a9e9a4f.png)

For this viewport size, I'd suggest both buttons have the same width, either `wide` or not, but not one and one.